content.md

```diff
@@ -43,38 +43,38 @@ We could use the Ruby [File](https://ruby-doc.org/3.2.1/File.html) or [CSV](http
+Relational databases, the kind of database that powers most apps, use a special language called Structured Query Language (SQL). Any app (whether Ruby, Python, Java, etc) that wants to store data in and retrieve data from a relational database must ultimately output instructions written in SQL (called "queries") and send them to the database for processing.
 
-In particular, we're going to use a very powerful and popular open-source database known as PostgreSQL (or "Postgres", for short). We're first going to see how to work with Postgres directly, but we'll soon move on to working with Postgres through Ruby. That will enable us to integrate permanent storage into our Rails apps – finally! Let's get started.
+In particular, we're going to use a very powerful and popular open-source database known as PostgreSQL (or "Postgres", for short). We're first going to see how to work with Postgres directly, but we'll soon move on to working with Postgres through Ruby. That will enable us to integrate permanent storage into our Rails apps – finally!
 
 - Select all that are true:
-- Structured Query Language (SQL) is only used for Ruby apps.
-  - Not quite, re-read the previous section.
-- SQL is the language of databases.
+- Structured Query Language (SQL) is the language used to interact with relational databases.
   - Yes!
-- Databases are part of a Rails app and can't be used by other types of apps.
-  - Not quite, re-read the previous section.
-- Databases are a piece of software separate from our app.
+- SQL is only used by Ruby apps.
+  - Not quite. Any app written in any language can use SQL to interact with a relational database.
+- The database is a separate app, and our Rails app connects to it when it wants to store or retrieve data.
   - Yes!
-- Our apps provide an interface to interact with a database.
+- The apps we build are essentially a way for users to interact with a database.
   - Yes!
-{: .choose_all #databases_and_sql title="Databases and SQL" points="3" answer="[2,4,5]" }
+{: .choose_all #databases_and_sql title="Databases and SQL" points="3" answer="[1,3,4]" }
 
 ## Working directly with Postgres
 
+<div class="alert alert-info">
 Don't worry about memorizing any of the raw Postgres and SQL commands that you'll see in this section. I just want you to have a sense of how the underlying database works before we graduate to using a wonderful Ruby gem called ActiveRecord that makes it all much easier.
+</div>
 
-First, you have to install the database that you want to use on your computer. Postgres is already installed on our codespaces, so we're set.
+First, you have to install the database that you want to use on your computer. Postgres is already installed on our Codespace, so we're set.
```

Comment:
```suggestion
First, you have to install the database that you want to use on your computer. Postgres is already installed on our codespace, so we're set.
```

I think we've been sticking with lowercase, unless referring to the product "GitHub Codespaces"